### ISSUE TITLE
Play one shot does not have option to exclude from reverb zone.

### ISSUE THREAD

**Issue by SUPTECGourry** [Yellow-Dog-Man/Resonite-Issues#250]
### Is your feature request related to a problem? Please describe.

Feature parity between flux node play oneshot and audio output component, to have a bool option to exclude the play oneshot from reverb zone.

### Describe the solution you'd like

Bool toggle on play oneshot to exclude the node audio from the reverb zone.

### Describe alternatives you've considered

The workaround I have to do is to mix flux and audio output and playback components to go around this limitation instead of a single node.

**Comment by Redd56** [comment]
yeah after reading fully again today this one seems very completed as they asked for a bool to ignore the reverb zones and thats what we got!
